packages/form-validations/README.md

```diff
@@ -0,0 +1,100 @@
+  formFieldSelector: '[data-spirit-validate]',
+  errorClass: 'has-error',
+  successClass: 'has-success',
+  errorTextParentSelector: '[data-spirit-validate]',
+  errorTextTag: 'div',
+  errorTextClass: '',
+  dataElementMessage: 'validation-text',
+};
+```
+
+- **live** - A boolean value indicating whether FormValidations should validate as you type, default is `true`
+
+## Built-in validators
+
+| Name      | Usage                                                                                                           | Description                     |
+| --------- | --------------------------------------------------------------------------------------------------------------- | ------------------------------- |
+| required  | `required` or `data-form-validations-required`                                                                  | Validates required fields       |
+| email     | `type="email"` or `data-form-validations-type="email"`                                                          | Validates email                 |
+| number    | `type="number"` or `data-form-validations-type="number"`                                                        |                                 |
+| integer   | `data-form-validations-type="integer"`                                                                          |                                 |
+| minlength | `minlength="10"` or `data-form-validations-minlength="10"`                                                      |                                 |
+| maxlength | `maxlength="10"` or `data-form-validations-maxlength="10"`                                                      |                                 |
+| min       | `min="20"` or `data-form-validations-min="20"`                                                                  |                                 |
+| max       | `max="100"` or `data-form-validations-max="100"`                                                                |                                 |
+| pattern   | `pattern="/[a-z]+$/i"` or `data-form-validations-pattern="/[a-z]+$/i"`, `\` must be escaped (replace with `\\`) |                                 |
+| equals    | `data-form-validations-equals="#field-selector"`                                                                | Check that two fields are equal |
+
+## API
+
+**FormValidations(form, config, live)**
+
+_Constructor_
+
+| Parameter | Default                      | Required? | Description                                         |
+| --------- | ---------------------------- | --------- | --------------------------------------------------- |
+| form      | -                            | ✔         | The form element                                    |
+| config    | [See above](#default-config) | ✕         | The config object                                   |
+| live      | `true`                       | ✕         | Whether FormValidations should validate as you type |
```

Comment:
Nice check marks! We could use them in all of our API tables.

Reply:
![](https://i.redd.it/hwurhp7crzf81.png)